src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/Datagrid.js

```diff
@@ -132,7 +132,7 @@ export default class Datagrid extends React.Component<Props> {
             <Fragment>
                 <div className={datagridStyles.toolbar}>
                     {searchable &&
-                        <Search onSearch={this.handleSearch} />
+                        <Search onSearch={this.handleSearch} value={store.searchTerm ? store.searchTerm.get() : null} />
```

Comment:
Is this check really necessary? From what I see `searchTerm` is always defined. Only the return value of `store.searchTerm.get()` could be undefined, but I think that would also be an acceptable option here.